### PR TITLE
chore: cleanup release-latest-versions

### PR DESCRIPTION
**Pull request by szepeviktor**
Please consider using https://github.com/koalaman/shellcheck
It is a single binary.

### CODE REVIEW

File: release-latest-versions.sh [hunk 1]
```diff
@@ -34,7 +34,6 @@ done
 do_release() {
 	local VERSION="$1"
 	local REPACK_VERSION="$2"
-	local SED_EXP
```
Comment (szepeviktor) [inline release-latest-versions.sh:37]: Unused

File: release-latest-versions.sh [hunk 2]
```diff
@@ -48,11 +47,11 @@ do_release() {
 
 	echo "Releasing v${VERSION}..."
 
-	# replace the version of wpackagist-plugin/wp-graphql in composer.json with $VERSION
-	contents="$(jq --arg version $VERSION '.require."wpackagist-plugin/wp-graphql" = $version' < source/composer.json)" && \
-	echo -E "${contents}" > source/composer.json
+	# Replace the version of wpackagist-plugin/wp-graphql in composer.json with $VERSION
+	contents="$(jq --arg version "${VERSION}" '.require."wpackagist-plugin/wp-graphql" = $version' < source/composer.json)" \
```
Comment (szepeviktor) [inline release-latest-versions.sh:51]: Make it safe and indent line

File: release-latest-versions.sh [hunk 3]
```diff
@@ -48,11 +47,11 @@ do_release() {
...
+		&& echo "${contents}" > source/composer.json
```
Comment (szepeviktor) [inline release-latest-versions.sh:52]: Make it safe and indent line

File: release-latest-versions.sh [hunk 4]
```diff
@@ -48,11 +47,11 @@ do_release() {
...
 
-	echo "$(cat source/composer.json)"
```
Comment (szepeviktor) [inline release-latest-versions.sh:55]: Useless echo

File: release-latest-versions.sh [hunk 5]
```diff
@@ -123,18 +122,17 @@ for POSSIBLE_VERSION in ${POSSIBLE_VERSIONS}; do
...
 
-		# 
 		if [ -z "${SHOULD_RERELEASE}" ]; then
 			echo "Skipping"
 			continue
 		fi
 
 		# If SHOULD_RERELEASE is true, append -r1, -r2, etc.
-		for i in {1..100}; do
+		for REPACK in {1..100}; do
```
Comment (szepeviktor) [inline release-latest-versions.sh:131]: Please see https://github.com/szepeviktor/uninformative-words

**Review by justlevine — APPROVED** [review]

**Pull request merged**